app/dashboard/templates/dashboard/sponsors.html

```diff
@@ -256,12 +257,19 @@
                                         Winner
                                       </b-form-checkbox>
                                     </b-td>
+                                    <b-td :class="isMobileDevice ? 'no-border mt-2': 'py-4'">
```

Comment:
how does is `isMobileDevice` btw ? when I shrink my screen -> none of these come into effect

Reply:
Thanks, fixed now :+1: